Cargo.toml

```diff
@@ -12,6 +12,11 @@ A library to run the pkg-config system tool at build time in order to be used in
 Cargo build scripts.
 """
 keywords = ["build-dependencies"]
+rust-version = "1.40"
+edition = "2018"
+
+[dependencies]
+regex = "1.6.0"
```

Comment:
`regex` is a very heavyweight dependency, and `pkg-config` is in the build phase of a huge number of crates.

This will likely slow down builds for these crates (and nearly the whole ecosystem as a result), so I would really prefer that this not be done using regexes.

The matching needed is not very sophisticated anyway, so the small amount of boilerplate this would add seems very much worth it to me.